Sources/KingfisherOptionsInfo.swift

```diff
@@ -84,13 +84,17 @@ public enum KingfisherOptionsInfoItem {
     /// It is the image scale, instead of your screen scale. You may need to specify the correct scale when you dealing 
     /// with 2x or 3x retina images.
     case scaleFactor(CGFloat)
-    
-    /// Whether all the GIF data should be preloaded. Default it false, which means following frames will be
-    /// loaded on need. If true, all the GIF data will be loaded and decoded into memory. This option is mainly
+
+    /// Whether all the animated image data should be preloaded. Default it false, which means following frames will be
+    /// loaded on need. If true, all the animated image data will be loaded and decoded into memory. This option is mainly
     /// used for back compatibility internally. You should not set it directly. `AnimatedImageView` will not preload
     /// all data, while a normal image view (`UIImageView` or `NSImageView`) will load all data. Choose to use
     /// corresponding image view type instead of setting this option.
-    case preloadAllGIFData
+    case preloadAllAnimationData
+
+    /// Back compatibility for deprecated preloadAllGIFData
+    @available(*, deprecated, renamed: "preloadAllAnimation")
```

Comment:
Here you are using a static property to "simulate" the syntax of an enum case:

```
@available(*, deprecated, renamed: "preloadAllAnimation")
static let preloadAllGIFData = KingfisherOptionsInfoItem.preloadAllAnimationData
```

So it is possible to move it to extension (it is not an enum member anymore). Sure you could keep it as an enum member (in which case you cannot move it to an extension), but I think it is a clever way to make it a static property here.

In there, this should also compile:

```swift
public extension KingfisherOptionsInfoItem {
    @available(*, deprecated, renamed: "preloadAllAnimation")
static let preloadAllGIFData = KingfisherOptionsInfoItem.preloadAllAnimationData
}
```

And users could get a rename warning if it is used.